sql/changes/LOADORDER

```diff
@@ -44,4 +44,5 @@
 1.5/issue-2380-modify-tax_extended_entry_id-constraint.sql
 1.5/issue-2278-modify-acc_trans-index.sql
 1.5/template_menu3.sql
+1.5/issue-2800-cant-delete-unused-account-after-books-closed.sql
```

Comment:
No need to create an `sql/changes/` item for this one: modules are always reloaded.
`sql/changes/` only comes in when changing the schema definition (adding/removing indexes, adding/changing/removing columns and tables, etc).